pkg/services/store/entity/sqlstash/folder_support.go

```diff
@@ -40,11 +40,11 @@ func updateFolderTree(ctx context.Context, tx *session.SessionTx, namespace stri
 		return err
 	}
 
-	query := "SELECT guid,uid,folder,name,slug" +
+	query := "SELECT guid,name,folder,name,slug" +
 		" FROM entity" +
-		" WHERE group=? AND resource=? AND namespace=?" +
+		" WHERE 'group'=? AND resource=? AND namespace=?" +
```

Comment:
maybe the column should be "group_name" so we don't have to worry about escaping it?

Reply:
I would suggest using [dialect.Quote()](https://github.com/grafana/grafana/blob/0424d44b39235c228ce0f2988d81a563dd1cf4ed/pkg/services/sqlstore/migrator/dialect.go#L19) to make sure that it works for all the backends.